dkpro-core-opennlp-asl/pom.xml

```diff
@@ -32,6 +32,13 @@
   <properties>
     <opennlp.version>1.9.0</opennlp.version>
   </properties>
+  <repositories>
```

Comment:
Please do not commit the staging repo in the POM. Better add it to your local `settings.xml` file to avoid having to copy/paste it all the time.